Gemfile.lock

```diff
@@ -401,7 +401,7 @@ GEM
     multi_json (1.15.0)
     multi_xml (0.6.0)
     nio4r (2.5.8)
-    nokogiri (1.13.8)
+    nokogiri (1.13.9)
```

Comment:
why? 
should be handled by https://github.com/openfoodfoundation/openfoodnetwork/pull/9832 ...